### PR TITLE
i18n: Dutch

### PR DESCRIPTION
**Pull request by Alkarex**
https://github.com/FreshRSS/FreshRSS/issues/949
https://github.com/Wanabo/FreshRSS-Dutch-translation

### CODE REVIEW

File: app/i18n/nl/gen.php [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+			'title_new_articles' => 'FreshRSS: nieuwe artikelen!',
+		),
+		'new_article' => 'Er zijn nieuwe artikelen beschikbaar, klik om de pagina te vernieuwen.',
+		'should_be_activated' => 'JavaScript moet aan staan',
+	),
+	'lang' => array(
+		'de' => 'Deutsch',
+		'en' => 'English',
+		'fr' => 'Français',
+		'nl' => 'Nederlands',
```
Comment (marienfressinaud) [inline app/i18n/nl/gen.php:123]: Note: we should add this line to other files + gen.lang.cz (Čeština) is missing here.

File: app/i18n/nl/conf.php [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+		),
+		'language' => 'Taal',
+		'notif_html5' => array(
+			'seconds' => 'seconden (0 betekent geen stop)',
+			'timeout' => 'HTML5 notificatie stop',
+		),
+		'theme' => 'Thema',
+		'title' => 'Opmaak',
+		'width' => array(
+			'content' => 'Content width',
```
Comment (marienfressinaud) [inline app/i18n/nl/conf.php]: Is it Dutch?

Comment (Alkarex) [inline app/i18n/nl/conf.php]: @Wanabo

File: app/i18n/nl/gen.php (hunk 1 above)
Comment (Alkarex) [inline app/i18n/nl/gen.php:123]: Done https://github.com/Alkarex/FreshRSS/commit/501e3312101893b147767c9f3a36c947869d6580
Maybe in the future we could extract this data, which is the same for all languages.

File: app/i18n/nl/gen.php [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+		'import' => 'Importeren',
+		'manage' => 'Beheren',
+		'mark_read' => 'Markeer als gelezen',
+		'mark_favorite' => 'Markeer als favoriet',
+		'remove' => 'Verwijder',
+		'see_website' => 'Bekijk website',
+		'submit' => 'Opslaan',
+		'truncate' => 'Verwijder alle artikelen',
+	),
+	'auth' => array(
```
Comment (Alkarex) [inline app/i18n/nl/gen.php:23]: This section does not seem to be the same structure as other languages.

**Comment by Wanabo** [comment]
Yes it is Dutch (Nederlands). I'll doublecheck the file with the remark about "This section does not seem to be the same structure as other languages.". But as far as I can tell now it is original only English is translated to Dutch. My site at http://www.nieuwskop.be (version 1.1.2.beta) is working perfectly with it. 

By the way, I'm also the translator for the Dutch languages files of the open source CMS PHP-Fusion.

Please keep in mind that I have two versions of Dutch translations available at github.
One for version 1.0.0 and one for version 1.2.2.beta

Suggestion for detecting available languages: Scan app\i18n\ for available languages and match them with a predefined table for full language display.

File: app/i18n/nl/gen.php (hunk 3 above)
Comment (Wanabo) [inline app/i18n/nl/gen.php:23]: Can't find anything wrong here.

Comment (Alkarex) [inline app/i18n/nl/gen.php:23]: Thanks for the feedback. I was referring to the `auth` section:
- en: https://github.com/Alkarex/FreshRSS/blob/Dutch/app/i18n/en/gen.php#L23
- nl: https://github.com/Alkarex/FreshRSS/blob/Dutch/app/i18n/nl/gen.php#L23

**Comment by Wanabo** [comment]
Couldn't find anything wrong in gen.php.
Missed a translation in conf.php
            'content' => 'Content width',
adjusted to:
            'content' => 'Inhoud breedte',

**Comment by Alwaysin** [comment]
Completely unrelated but @Wanabo, could you share your CSS rules?

I love the density you achieved on your website.

**Comment by Alkarex** [comment]
> Missed a translation in conf.php

Done https://github.com/Alkarex/FreshRSS/commit/b3ae9407aca2223000c4f3a5b56648382de8ef48

> Couldn't find anything wrong in gen.php.

See in `auth`: `password, registration, username`, which should be arrays (maybe @marienfressinaud can confirm the intention) :
- en: https://github.com/Alkarex/FreshRSS/blob/Dutch/app/i18n/en/gen.php#L23
- nl: https://github.com/Alkarex/FreshRSS/blob/Dutch/app/i18n/nl/gen.php#L23

It is also good to double-check that `en/gen.php` and `nl/gen.php` have the same number of lines.

**Comment by Wanabo** [comment]
I've rechecked every file for version 1.1.2 beta.

I updated the files when I saw non translated parts on my website when I updated from version 1.0.0. to 1.1.2 beta. That is not a good strategy! It seems I missed some additions. Now at least all files have the same number of lines and did a file comparison between 'en' and 'nl'.

I've noticed a lot of language definitions return in different files. Isn't it possible to define returning strings in a master language file. That would save work. For example install.php and admin.php share a lot of content.

Or an on board language editor.

**Comment by marienfressinaud** [comment]
@Alkarex > you should consider branch 1.1.2 of @Wanabo: https://github.com/Wanabo/FreshRSS-Dutch-translation/tree/1.1.2-beta

**Comment by Alkarex** [comment]
@marienfressinaud Yes, that is the one I took ( https://github.com/Wanabo/FreshRSS-Dutch-translation/blob/c2385d656f598ae8344506e67337bad4e8dd3dac/gen.php ) I can see that there has been some changes today.

**Comment by Alkarex** [comment]
I have merged today's changes from https://github.com/Wanabo/FreshRSS-Dutch-translation/tree/1.1.2-beta : https://github.com/Alkarex/FreshRSS/commit/b33004c38145d4f4c0d2532e3832743b8ad5ba27

**Comment by Wanabo** [comment]
I've rechecked every file for version 1.1.2 beta.

I updated the files when I saw non translated parts on my website when I updated from version 1.0.0. to 1.1.2 beta. That is not a good strategy! It seems I missed some additions. Now at least all files have the same number of lines and did a file comparison between 'en' and 'nl'.

I've noticed a lot of language definitions return in different files. Isn't it possible to define returning strings in a master language file. That would save work. For example install.php and admin.php share a lot of content.

Or an on board language editor.

**Comment by Wanabo** [comment]
@Alwaysin It's very simple. I added some modified css from base-theme to dark.css and left base-theme original. You can download Theme DarkDense (inspired on Theme Dark) here, http://www.nieuwskop.be/theme-upload/DarkDense.zip

**Comment by Alwaysin** [comment]
Thanks for the theme! Works great :+1: 

Maybe we could add it in FreshRSS by default?

**Comment by marienfressinaud** [comment]
@Alkarex > is the PR ready to merge?

@Wanabo and @Alwaysin > note there is [an extension](https://github.com/FreshRSS/Extensions) to add CSS rules: I think you don't need a specific theme to achieve this ;)

**Comment by Alkarex** [comment]
@marienfressinaud Yes, ready. The PR integrates the latest changes from https://github.com/Wanabo/FreshRSS-Dutch-translation/commits/1.1.2-beta

**Comment by Wanabo** [comment]
Yeah with xExtension-CustomCSS all themes become dense with just a few lines of CSS.

I've setup a small tutorial. <a href="http://www.webmastertrouble.com/forum/viewthread.php?thread_id=43">Achieve more dense themes with xExtension-CustomCSS</a>

**Comment by marienfressinaud** [comment]
Merged!

Thank you @Wanabo :)

**Pull request merged**